test/topology_custom/test_alternator.py

```diff
@@ -65,8 +65,8 @@ async def alternator3(manager_internal):
        Returns the manager object and 3 boto3 resource objects for making
        DynamoDB API requests to each of the nodes in the Alternator cluster.
     """
-    manager = manager_internal()
-    servers = await manager.servers_add(3, config=alternator_config)
+    manager = manager_internal
+    servers = [await manager.server_add(config=alternator_config) for _ in range(3)]
```

Comment:
this patch should be folded into the backport itself and the other patch which disables tablets should be the first patch, so we preserve bisectability

Reply:
This is a matter of style. I actually though it is better to keep the automatic merge separately and add stuff on top of it.

I'll redo this PR tomorrow morning yet again, but the inefficiency of our development process is getting almost comical. I think you wrote this 3-line patch a month ago already, for a P1 issue, and look at how much time it's taking to get it in :-(

Reply:
Done